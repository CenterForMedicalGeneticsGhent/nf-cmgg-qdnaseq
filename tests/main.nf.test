nextflow_pipeline {

    name "Test Workflow main.nf"
    script "main.nf"
    options "-stub"

    test("Should run without failures") {

        expect {
            def date = new java.text.SimpleDateFormat("yyyy-MM-dd").format(new Date())
            assertAll(
                { assert workflow.success },
                { assert snapshot(
                        path("${outputDir}/annotations-${date}/hg38.5kbp.rda"),
                        path("${outputDir}/annotations-${date}/hg38.10kbp.rda"),
                        file("${outputDir}/multiqc_reports/multiqc_report.html").exists()
                    ).match()
                }
            )
        }

    }

}
