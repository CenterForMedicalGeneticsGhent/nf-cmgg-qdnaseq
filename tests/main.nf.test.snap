{
    "Should run without failures": {
        "content": [
            "hg38.5kbp.rda:md5,d41d8cd98f00b204e9800998ecf8427e",
            "hg38.10kbp.rda:md5,d41d8cd98f00b204e9800998ecf8427e",
            true
        ],
        "timestamp": "2023-07-04T09:02:29+0000"
    }
}